File: Level1/약수의 개수와 덧셈.py
def solution(left, right):
    sum = 0
    for num in range(left, right + 1):
        cnt = 1
        for i in range(1,num//2 + 1):
            if num%i==0:
                cnt += 1
        if cnt%2==0:
            sum += num
        else:
            sum -= num
    return sum

File: Level1/약수의 개수와 덧셈.js
function solution(left, right) {
    let sum = 0;
    for (let n = left; n <= right; n++) {
        if (Number.isInteger(Math.sqrt(n))) {
            sum -= n;
        } else {
            sum += n;
        }
    }
    return sum;
}

//제곱근이 정수면 약수의 개수가 홀수이다.
